tests/integration/test_bundle_deployment.py

```diff
@@ -9,41 +9,55 @@
 from pytest_operator.plugin import OpsTest
 from lightkube.resources.core_v1 import Service
 
+from _pytest.config.argparsing import Parser
+
 BUNDLE_NAME = "kubeflow"
 
 @pytest.fixture()
 def lightkube_client() -> lightkube.Client:
     client = lightkube.Client(field_manager=BUNDLE_NAME)
     return client
 
+
 @pytest.fixture
-def bundle_path() -> str:
-    return os.environ.get("BUNDLE_PATH").replace("\"", "")
+def bundle_path(request) -> str:
+    """Return the relative path to the bundle file given a track and risk."""
+    track = request.config.getoption("track")
+    risk = request.config.getoption("risk")
+    bundle_path = sh.find("./releases/{track}/{risk}", "-name", "bundle.yaml")
```

Comment:
I ran this locally for latest/edge
```
╰─$ find ./releases/latest/edge -name bundle.yaml
./releases/latest/edge/build/parts/bundle/install/bundle.yaml
./releases/latest/edge/build/parts/bundle/build/bundle.yaml
./releases/latest/edge/build/parts/bundle/src/bundle.yaml
./releases/latest/edge/build/stage/bundle.yaml
./releases/latest/edge/build/prime/bundle.yaml
./releases/latest/edge/bundle.yaml
```

Reply:
I believe this is because you have built the bundle and the `build` directory was added. Try removing it from your tree. If you check the repo, these directories and bundles shouldn't be there.